braindecode/datautil/transforms.py

```diff
@@ -77,103 +75,20 @@ def _transform(raw_or_epochs, transforms):
             getattr(raw_or_epochs.load_data(), transform)(**transform_kwargs)
 
 
-class FilterRaw(object):
-    """Apply mne filter on raw data
-
-    Parameters
-    ----------
-    mne_filter_kwargs : **kwargs
-        kwargs passed to mne.io.Raw.filter
-    """
-
-    def __init__(self, **mne_filter_kwargs):
-        self.mne_filter_kwargs = mne_filter_kwargs
-
-    def __call__(self, raw):
-        """Apply filter
-
-        Parameters
-        ----------
-        raw : mne.io.Raw
-            raw data to filter
-        """
-        return raw.filter(**self.mne_filter_kwargs)
-
-
-class ZscoreRaw(object):
-    """Zscore raw data channel wise
-    """
-
-    def __call__(self, raw):
-        """Zscore Normalize raw data channel wise
-
-        Parameters
-        ----------
-        raw : mne.io.Raw
-            raw data to normalize
-
-        Returns
-        -------
-        raw : mne.io.Raw
-            normalized raw data
-
-        """
-        raw = raw.apply_function(lambda x: x - x.mean())
-        return raw.apply_function(lambda x: x / x.std())
-
-
-class FilterWindow(object):
-    """FIR filter for windowed data.
+def zscore(continuous_data):
+    """Zscore raw data
 
     Parameters
     ----------
-    sfreq : int | float
-        sampling frequency of data when applying filter
-    l_freq : int | float | None
-        see mne.filter.create_filter
-    h_freq : int | float | None
-        see mne.filter.create_filter
-    kwargs : **kwargs
-        see mne.filter.create_filter
-    overlap_kwargs  : **kwargs
-        see mne.filter._overlap_add_filter
-    """
-
-    def __init__(
-        self, sfreq, l_freq=None, h_freq=None, kwargs=None, overlap_kwargs=None
-    ):
-        if kwargs is None:
-            kwargs = dict()
-        if overlap_kwargs is None:
-            overlap_kwargs = dict()
-        self.filter = mne.filter.create_filter(
-            None, sfreq, l_freq=l_freq, h_freq=h_freq, method="fir", **kwargs
-        )
-        self.overlap_kwargs = overlap_kwargs
-
-    def __call__(self, windows):
-        return mne.filter._overlap_add_filter(
-            windows, self.filter, **self.overlap_kwargs
-        )
-
+    continuous_data: np.ndarray
```

Comment:
For completeness, should we add the expected dimensions (n_channels, n_times)?

Reply:
With the changes discussed above it could then also ne (n_windows x n_channels x n_times). I will add this.